### ISSUE TITLE
Enable livepatch on Xenial+

### ISSUE THREAD

**Issue by powersj** [CanonicalLtd/ubuntu-advantage-client#679]
Bug to track the enablement of livepatch on Bionic and Xenial

~- [ ] Get Xenial GPG Key~
~- [ ] Get Bionic GPG Key~
- [ ] Ensure enable installs correct packages 
- [ ] Ensure disable correctly works
- [ ] Ensure enable continues to work on Trusty
- [ ] Ensure status works as root
- [ ] Ensure status works as non-root
- [ ] Ensure enable works on generic kernel
- [ ] Ensure enable works on aws kernel
- [ ] Get complete list of other places enable should work and update this list

**Comment by blackboxsw** [comment]
```
#### start Xenial
multipass@dev-x:~$ sudo ua attach <REDACTED>
Enabling default service esm
Enabling authenticated repo: https://esm.ubuntu.com
Updating package lists
Extended Security Maintenance enabled.
Enabling default service livepatch
Installing canonical-livepatch snap
Canonical livepatch enabled.
This machine is now attached to 'UA Client Test Contract'.

                Account: UA Client Test
           Subscription: UA Client Test Contract
            Valid until: n/a
Technical support level: advanced

SERVICE
cc-eal        entitled         inactive
cis-audit     none             n/a
esm           entitled         active
fips          entitled         inactive
fips-updates  entitled         inactive
livepatch     entitled         active

Enable entitlements with `ua enable <service>`


#### start Bionic

**Comment by blackboxsw** [comment]
# begin trusty
multipass@dev-t2:~$ sudo ua status
                Account: UA Client Test
           Subscription: UA Client Test Contract
            Valid until: n/a
Technical support level: advanced

SERVICE
cc-eal        entitled         n/a
cis-audit     none             n/a
esm           entitled         active
fips          entitled         n/a
fips-updates  entitled         n/a
livepatch     entitled         n/a

Enable entitlements with `ua enable <service>`
multipass@dev-t2:~$ uname -r
3.13.0-170-generic
multipass@dev-t2:~$ sudo ua enable livepatch
One moment, checking your subscription first
Livepatch is not available for kernel 3.13.0-170-generic.
Minimum kernel version required: 4.4
multipass@dev-t2:~$ sudo apt-get install --install-recommends linux-generic-lts-xenial 
Reading package lists... Done
...
multipass@dev-t2:~$ sudo reboot
multipass@dev-t2:~$ sudo ua enable livepatch
One moment, checking your subscription first
Installing canonical-livepatch snap
Canonical livepatch enabled.
multipass@dev-t2:~$ sudo ua status
                Account: UA Client Test
           Subscription: UA Client Test Contract
            Valid until: n/a
Technical support level: advanced

SERVICE
cc-eal        entitled         n/a
cis-audit     none             n/a
esm           entitled         active
fips          entitled         n/a
fips-updates  entitled         n/a
livepatch     entitled         active

Enable entitlements with `ua enable <service>`
multipass@dev-t2:~$ sudo /snap/bin/canonical-livepatch status
client-version: 9.4.1
architecture: x86_64
cpu-model: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
last-check: 2019-07-13T11:02:04-06:00
boot-time: 2019-07-13T10:37:06-06:00
uptime: 25m15s
status:
- kernel: 4.4.0-154.181~14.04.1-generic
  running: true
  livepatch:
    checkState: checked
    patchState: nothing-to-apply
    version: ""
    fixes: ""

multipass@dev-t2:~$ sudo ua disable livepatch
Removing canonical-livepatch snap
snap list
Livepatch disabled.
multipass@dev-t2:~$ snap list
Name  Version    Rev   Tracking  Publisher   Notes
core  16-2.39.3  7270  stable    canonical✓  core
